### ISSUE TITLE
Mixed-case or uppercase variables are not replaced

### ISSUE THREAD

**Issue by yissachar** [kopeio/kexpand#2]
Running `kexpand` on this template:

```
apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  name: hello-world
spec:
  replicas: $((numReplicas))
  template:
    metadata:
      labels:
        app: hello-world
    spec:
      containers:
      - name: hello-world
        image: tutum/hello-world
        ports:
        - containerPort: 80
```

Does not result in any errors, but the variable is left untouched. Changing the variable to `numreplicas` works as expected.

**Comment by denniswebb** [comment]
Shouldn't this be closed as a result of #4 ?

**Comment by justinsb** [comment]
I believe so - thanks for #4 - closing!
